src/main/java/org/fundacionjala/sfdc/Library.java

```diff
@@ -1,5 +1,10 @@
 package org.fundacionjala.sfdc;
 
+import org.fundacionjala.sfdc.pageobjects.Accounts.SFANewModifyPage;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.pageobjects.Accounts.SFANewModifyPage'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)